mucommander-core/src/main/java/com/mucommander/ui/action/impl/OpenTrashAction.java

```diff
@@ -64,6 +68,8 @@ public static class Descriptor extends AbstractActionDescriptor {
 
 		public KeyStroke getDefaultAltKeyStroke() { return null; }
 
-		public KeyStroke getDefaultKeyStroke() { return null; }
+		public KeyStroke getDefaultKeyStroke() {
+            return KeyStroke.getKeyStroke(KeyEvent.VK_DELETE, KeyEvent.ALT_MASK);
+        }

```

Comment:
something about the layout here is wrong, please apply the[ code formatter](https://github.com/mucommander/mucommander/blob/master/mucommander-code-format.xml)?

Reply:
ooops, this sadly results from mixed tabs and spaces in the project. I will fix that (I will create a separate PR for it)

btw, maybe all the files in the project should be updated to replace tabs with spaces (tabs confuse Idea sometimes even if spaces are set as default tabulation char).

Reply:
moved to #616

Reply:
> btw, maybe all the files in the project should be updated to replace tabs with spaces (tabs confuse Idea sometimes even if spaces are set as default tabulation char).

yeah, the convention has changed over the years but changing everything now has some downside like looking the ability to quickly find the last change that affected a line with 'blame'.. not sure this would worth the effort

Reply:
maybe around version 1.0.0 (a mile stone :)) it is worth to take a risk - do bulk formatting, it can be easier to have more refactoring and features to be implemented? Just thinking out loud...

btw, what's the lowest java version muC should support?